packages/backend/src/modules/activity/counters/StarkexCounter.test.ts

```diff
@@ -2,37 +2,33 @@ import { Logger } from '@l2beat/backend-tools'
 import { ProjectId, UnixTime } from '@l2beat/shared-pure'
 import { install } from '@sinonjs/fake-timers'
 import { expect, mockObject } from 'earl'
-import { Knex } from 'knex'
 import waitForExpect from 'wait-for-expect'
 
+import { Database, Transaction } from '@l2beat/database'
 import { StarkexClient } from '../../../peripherals/starkex/StarkexClient'
 import { Clock } from '../../../tools/Clock'
-import { SequenceProcessorRepository } from '../repositories/SequenceProcessorRepository'
-import { StarkexTransactionCountRepository } from '../repositories/StarkexCountRepository'
 import { StarkexCounter } from './StarkexCounter'
 
 describe(StarkexCounter.name, () => {
   it('iterates over products', async () => {
     const project = ProjectId('starkex_project')
-    const starkexRepository = mockObject<StarkexTransactionCountRepository>({
-      addOrUpdateMany: async () => -1,
-    })
     const txCount = 1
     const starkexClient = mockObject<StarkexClient>({
       getDailyCount: async () => txCount,
     })
-    const transaction = mockObject<Knex.Transaction>({})
-    const sequenceProcessorRepository = mockObject<SequenceProcessorRepository>(
-      {
+    const starkExTransactionCount = mockObject<
```

Comment:
Why is this mock defined outside the `db` object below?

Reply:
Otherwise earl expects seem to be not working correctly. Decided it wasn't important enough to worry about